code/game/objects/structures/morgue.dm

```diff
@@ -354,12 +332,58 @@ GLOBAL_LIST_EMPTY(crematoriums)
 	add_fingerprint(user)
 	update_icon(UPDATE_OVERLAYS)
 
-/obj/structure/crematorium/attackby(P as obj, mob/user as mob, params)
+/obj/structure/crematorium/obj_break(damage_flag)
+	if(!broken)
```

Comment:
could you make this an early return please

Reply:
Might I ask what you mean here?